src/map/clif.cpp

```diff
@@ -5242,6 +5242,12 @@ int clif_damage(struct block_list* src, struct block_list* dst, t_tick tick, int
 		}
 	}
 
+	// Calculate what sdelay to send to the client so it applies damage at the same time as the server
+	if (src->type == BL_MOB && battle_config.synchronize_damage && status_get_clientamotion(src) != 0) {
```

Comment:
```suggestion
	if (battle_config.synchronize_damage && src->type == BL_MOB && status_get_clientamotion(src) != 0) {
```
Feature check first, so that it fails as fast as possible.